### PR TITLE
Added api1800 support for fc and fcoe network

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added api1800 support for fc and fcoe network

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [x] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [x] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: spec/unit/resources/fc_network/spec_delete_bulk.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require_relative './../../../spec_helper'
```
Comment (yuvirani) [inline spec/unit/resources/fc_network/spec_delete_bulk.rb:12]: Add copy rights in this file

File: spec/unit/resources/fc_network/spec_delete_bulk.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
```
Comment (yuvirani) [inline spec/unit/resources/fc_network/spec_delete_bulk.rb:13]: Same as above

File: spec/unit/resources/fc_network/spec_delete_bulk.rb (hunk 1 above)
Comment (VenkateshRavula) [reply spec/unit/resources/fc_network/spec_delete_bulk.rb:12]: none of chef spec files have copy rights mentioned

Comment (VenkateshRavula) [reply spec/unit/resources/fc_network/spec_delete_bulk.rb:12]: done

File: spec/unit/resources/fc_network/spec_delete_bulk.rb (hunk 2 above)
Comment (VenkateshRavula) [reply spec/unit/resources/fc_network/spec_delete_bulk.rb:13]: done

**Review by yuvirani — APPROVED** [review]

**Review by chebroluharika — APPROVED** [review]

**Pull request merged**